Sources/UInt128.swift

```diff
@@ -636,10 +636,7 @@ extension UInt128 : Numeric {
 extension UInt128 : Equatable {
     /// Checks if the `lhs` is equal to the `rhs`.
     public static func ==(lhs: UInt128, rhs: UInt128) -> Bool {
-        if lhs.value.lowerBits == rhs.value.lowerBits && lhs.value.upperBits == rhs.value.upperBits {
-            return true
-        }
-        return false
+        return lhs.value == rhs.value
```

Comment:
Would you mind profiling this change? At first glance my gut tells me it would be quite a bit slower, but maybe my gut is wrong.